GeneratorInterface/Pythia8Interface/plugins/Pythia8Hadronizer.cc

```diff
@@ -568,6 +579,13 @@ bool Pythia8Hadronizer::initializeForExternalPartons() {
     fMultiUserHook->addHook(fPowhegHooksBB4L.get());
   }
 
+  bool TopRecoilHook1 = fMasterGen->settings.flag("TopRecoilHook:doTopRecoilIn");
+  if (TopRecoilHook1) {
+    edm::LogInfo("Pythia8Interface") << "Turning on RecoilToTop hook from Pythia8Interface";
+    fTopRecoilHook.reset(new TopRecoilHook());
```

Comment:
In the master there is a check
```suggestion
    if (!fTopRecoilHook.get())
      fTopRecoilHook.reset(new TopRecoilHook());
```
Why is it not backported here?